src/tags/object/Audio.js

```diff
@@ -12,7 +12,7 @@ import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage";
 import { AnnotationMixin } from "../../mixins/AnnotationMixin";
 
 /**
- * Audio tag plays a simple audio file.
+ * Audio tag plays a simple audio file
```

Comment:
does the period break something?

Reply:
of course no :)
we can add it back for sure